docs/known_issues.rst

```diff
@@ -99,6 +99,13 @@ An easy solution is::
     array([False], dtype=bool)
 
 
+Quantities in np.linspace failure on numpy 1.10
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In numpy 1.10, `~numpy.linspace` initially failed to work with quantities.
```

Comment:
@mhvk - I would rephrase to say 'linspace does not work correctly with quantities when using Numpy 1.10.0 to 1.10.5 due to a bug in Numpy. The solution is to upgrade to Numpy 1.10.6, in which the bug was fixed'